# P.excavatus Regeneration Transcriptome Data

---

Atlas of the transcriptome dynamics during the bidirectional regeneration in Perionyx excavates

---


---

* e.g.
[File Name]
: Description

---


### 1. De novo assembly - Trinity(v2.1.1)

#### [Perionyx_Reg_Draft_Trinity.fasta]
: Draft de novo transcriptome assembly using Trinity(v2.1.1) (551,875 transcripts)   

> Unable to upload to github because the file size of the assembly is larger than 25MB.
> We will send it to you upon request by e-mail. (sunggwonl22@gmail.com)

---

### 2. Gene Prediction - TransDecoder(v3.0.0) & CD-HIT(v.4.6.5)

#### [Perionyx_Reg_Removal_Redundant_mRNA.fasta]
: mRNA sequences (5’UTR + CDS + 3’UTR) after Gene prediction and removing redundant   

> Unable to upload to github because the file size of the assembly is larger than 25MB.
> We will send it to you upon request by e-mail. (sunggwonl22@gmail.com)
 

#### [Perionyx_Reg_Removal_Redundant_prot.fasta]
: Coding sequences (amino acid) sequences after Gene prediction and removing redundant     

---

### 3. Expression level

#### [Perionyx_Reg_EXP.tsv]
: Expression level (TMM normalized FPKM) of 19 samples   

---

### 4. BUSCO analysis

#### [Perionyx_Reg_BUSCO_eukaryota.zip]
: Results of BUSCO analysis (Eukaryote single-copy orthologs)   

#### [Perionyx_Reg_BUSCO_metazoa.zip]
: Results of BUSCO analysis (Metazoan single-copy orthologs)   

---

### 5. Differentially expressed geens (DEGs) 

#### [Perionyx_Reg_DEGs.xlsx]
:the DEGs of bidirectional regeneration were clustered for head and tail specific during the regeneration by K-mean clustering(z-score).

---

### 6. Cluster analysis

#### [Perionyx_Reg_three_main_cluster.xlsx]
:Three main clusters (head or tail specific DEGs)


